resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row justify-content-center text-center">
+            <div class="bg-white border p-5 w-50 ml-5 mt-5">
+                <div class="row justify-content-center">
+                    <label class="h2 font-weight-bold text-uppercase">
+                        {{ trans('login.title') }}
+                    </label>
+                </div>
+                <hr>
+                <form method="POST" action="{{ route('login') }}">
+                    @csrf
+                    <div class="input-group input-group-lg p-2">
+                        <input type="email" class="form-control rounded-pill pl-4 @error('email') is-invalid @enderror" name="email" placeholder="{{ trans('login.email') }}"  value="{{ old('email') }}" required autocomplete="email" autofocus>
```

Comment:
```suggestion
                        <input type="email" class="form-control rounded-pill pl-4 @error ('email') is-invalid @enderror" name="email" placeholder="{{ trans('login.email') }}"  value="{{ old('email') }}" required autocomplete="email" autofocus>
```
sửa hết các lỗi tương tự bên dưới đi, có những chỗ tương tự t k cmts chứ k phải k lỗi nhé, trước khi ready xem lại 1 lượt

Reply:
Chị huyền nói là ko cần có space sau @error